lib/llm_eval_ruby/prompt_adapters/local.rb

```diff
@@ -7,14 +7,26 @@ module LlmEvalRuby
   module PromptAdapters
     class Local < Base
       class << self
+        # Versioning was introduced to allow flexibility in managing different iterations of prompts.
+        # This enables testing and gradual rollouts of updated prompt versions without affecting existing ones.
+        #
+        # lib/prompts/test
+        # ├── system.txt
+        # ├── user.txt
+        # └── v2
+        #     ├── system.txt
+        #     └── user.txt
+        # └── v3
+        #     ├── system.txt
+        #     └── user.txt
         def fetch_prompt(name:, version: nil) # rubocop:disable Lint/UnusedMethodArgument
-          prompt_path = Rails.root.join(LlmEvalRuby.config.local_options[:prompts_path], name.to_s)
+          prompt_path = Rails.root.join(LlmEvalRuby.config.local_options[:prompts_path], name.to_s, version.to_s)
 
-          system_prompts = Dir.glob("#{prompt_path}/**/system.txt").map do |path|
```

Comment:
are you sure about that?

Reply:
I mean I really don't know whats behind this tree structure defined by @aleksei-okatiev